docs/troubleshooting.md

```diff
@@ -22,11 +22,8 @@ to use the same versions.
 
 When a rebuild is not possible,
 for .NET Framework applications the workaround is to use binding redirects.
-The [examples/BindingRedirect](./../examples/BindingRedirect/) app shows how
-to use the `app.config` file to solve version conflicts.
-The example can only run successfully under the instrumentation, as the
-binding redirect makes the application dependent on a version of
-`System.Diagnostics.DiagnosticSource` that is not available at build time.
+The binding redirect makes the application dependent on a version
```

Comment:
The wording here is not technically correct. It redirects a range of versions to a specific version.

Reply:
:shipit: